main.js

```diff
@@ -0,0 +1,234 @@
+// const namesarray = ['secondimage', 'imagehovered'];
+// const descriptionarray = ['css style', 'js program'];
+// const readmore = ['readmore'];
+
+// const dynamic = document.querySelector('.gridcontainer2');
+// // eslint-disable-next-line no-plusplus
+// for (let i = 0; i < namesarray.length; i++) {
+//   const fetch = document.querySelector('.gridcontainer2').innerHTML;
+//   const bgimg = document.getElementById(`cards${i}`);
+//   dynamic.innerHTML = `<div id="cards${i}" class="boxes">
+//   <div class="box-content">
+//   <h2>${namesarray[i]}</h2>
+//   <p>  ${descriptionarray[i]} </p>
+//   <p> <a class="showmore" href="#">${readmore[0]}</a></p>
+//   <img ${bgimg} />
+//   </div>
+//   </div>${(fetch)}`;
+//   bgimg.style.backgroundImage = `url('./images/${namesarray[i]}.png)`;
+// }
```

Comment:
- [ ] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)